src/redist/targets/BundledManifests.targets

```diff
@@ -1,5 +1,6 @@
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
+    <NetFeatureBandPrerelease>8.0.100-$(PreReleaseVersionLabel).$(PreReleaseVersionIteration)</NetFeatureBandPrerelease>
```

Comment:
This isn't 100% correct since it is encoded in the package name and could be different from the sdk PreReleaseVersion* setting but I can't use the package version for the same reason and using the package name here would break intent of the prerelease setting. https://github.com/dotnet/installer/issues/15367

Reply:
At least this way it will fail if there isn't a package that satisfies the conditions required to make it mostly work